### PR TITLE
inventory: Update azure x64 dockerhost to ubuntu 2404 and update DockerInventory.json

### PR DESCRIPTION
**Pull request by Haroon-Khel**
- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [x] VPC/QPC not applicable for this PR
- [x] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

ref https://github.com/adoptium/infrastructure/issues/3693

I updated the docker inventory in the same pr. Alot of the changes detected by git in the DockerInventory.json file are line position changes which is confusing git, but its fine

### CODE REVIEW

File: ansible/DockerInventory.json [hunk 1]
```diff
@@ -328,6 +344,6 @@
                 "port": "32790"
             }
         ],
-        "containersCount": 4
+        "containersCount": 3
     }
 ]
```
Comment (sxa) [inline ansible/DockerInventory.json:349]: There should perhaps be a newline character on the end of this, but I'll approve regardless :-)

**Review by sxa — APPROVED** [review]

**Review by karianna — APPROVED** [review]

**Comment by karianna** [comment]
@Haroon-Khel feel free to merge after you look at sxa's comment :-)

**Review by steelhead31 — APPROVED** [review]
Looks ok.

File: ansible/DockerInventory.json (hunk 1 above)
Comment (Haroon-Khel) [reply ansible/DockerInventory.json:349]: The json file is generated by the https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/scripts/updateDockerStaticInventory.py script, so I can add a change in there to create a new line at the end of the file

**Pull request merged**